keyboards/ropro/keymaps/default/default.c

```diff
@@ -0,0 +1,76 @@
+/* Copyright 2015-2017 Jack Humbert
```

Comment:
This file should be named `keymap.c`.

Specifically, the build script is expecting `/keyboards/(keyboard)/keymaps/default/keymap.c` to be present.